.github/workflows/ks-review.yml

```
@@ -27,8 +27,8 @@ env:
   BUILD_ARTIFACT_FOLDER: 'build_artifacts'
   SERVICE_ARTIFACT_FOLDER: 'service_artifacts'
   BASE_DOMAIN: ${{ secrets.BASE_DOMAIN_STAGING }}
-  K8S_REQUEST_CPU: 50m
-  K8S_REQUEST_RAM: 256Mi
+  K8S_REQUEST_CPU: 5m
+  K8S_REQUEST_RAM: 32Mi
```

```suggestion
  K8S_REQUEST_RAM: 256Mi
```

No need to reduce, apps use currently 150-250 Mi of memory